Pricing & Settings/One Agent/LinkedTemplates/Solutions-deployIfNotExists.json

```diff
@@ -0,0 +1,73 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "workspaceName": {
+        "type": "string"
+      },
+      "location": {
+        "type": "string"
+      }
+    },
+    "variables": {
+      "securitySolution": {
+        "Name": "[Concat('Security', '(', parameters('workspaceName'), ')')]",
+        "GalleryName": "Security"
+      },
+      "securityCenterFreeSolution": {
+        "Name": "[Concat('SecurityCenterFree', '(', parameters('workspaceName'), ')')]",
+        "GalleryName": "SecurityCenterFree"
+      },
+      "antiMalwareSolution": {
+        "Name": "[Concat('AntiMalware', '(', parameters('workspaceName'), ')')]",
+        "GalleryName": "AntiMalware"
+      }
+    },
+    "resources": [
+      {
+        "type": "Microsoft.OperationsManagement/solutions",
+        "name": "[variables('securitySolution').Name]",
+        "location": "[parameters('location')]",
+        "apiVersion": "2015-11-01-preview",
+        "properties": {
+          "workspaceResourceId": "[resourceId('Microsoft.OperationalInsights/workspaces/', parameters('workspaceName'))]"
+        },
+        "plan": {
+          "name": "[variables('securitySolution').Name]",
+          "publisher": "Microsoft",
+          "product": "[Concat('OMSGallery/', variables('securitySolution').GalleryName)]",
+          "promotionCode": ""
+        }
+      },
+      {
+        "type": "Microsoft.OperationsManagement/solutions",
+        "name": "[variables('securityCenterFreeSolution').Name]",
+        "location": "[parameters('location')]",
+        "apiVersion": "2015-11-01-preview",
+        "properties": {
+          "workspaceResourceId": "[resourceId('Microsoft.OperationalInsights/workspaces/', parameters('workspaceName'))]"
+        },
+        "plan": {
+          "name": "[variables('securityCenterFreeSolution').Name]",
+          "publisher": "Microsoft",
+          "product": "[Concat('OMSGallery/', variables('securityCenterFreeSolution').GalleryName)]",
+          "promotionCode": ""
+        }
+      },
+      {
+        "type": "Microsoft.OperationsManagement/solutions",
```

Comment:
Anti malware MP is part of Security Solution - so I don't think we need to add it. @bmanikm  to confirm.

Reply:
I had to enable the Antimalware LA solution explicitly to query the protection status logs in LA portal.

---
In reply to: [439824893](https://github.com/Azure/Azure-Security-Center/pull/135#discussion_r439824893) [](ancestors = 439824893)

Reply:
@bmanikm , but since AM MP is part of Security this is redundant, no ? We don't use AM solution or enable it anywhere in ASC today. I would rather not direct customers to use it.

Reply:
@omerchechik, I agree with you, AM solution is not required. I have tried removing the AM solution from LA workspace and I can still query the protection status logs in LA portal.
@prash2611, please consider removing the AM from the ARM templates.

Reply:
Removed!